sdk/generic-sdk-core/generic-core/src/main/java/module-info.java

```diff
@@ -14,14 +14,14 @@
     exports com.generic.core.http.exception;
     exports com.generic.core.http.models;
     exports com.generic.core.http.pipeline;
-    exports com.generic.core.http.policy;
     exports com.generic.core.models.traits;
     exports com.generic.core.util;
     exports com.generic.core.util.binarydata;
     exports com.generic.core.util.configuration;
     exports com.generic.core.util.serializer;
 
     exports com.generic.core.implementation.util to com.generic.core.http.okhttp;
+    exports com.generic.core.implementation.http.policy;
```

Comment:
We don't need to export this package, I think IntelliJ added this to be safe.

Reply:
yeah didnt see this get added. We should be good without it.